tests/baselines/reference/inKeywordTypeguard(strict=true).errors.txt

```diff
@@ -240,8 +239,6 @@ tests/cases/compiler/inKeywordTypeguard.ts(186,21): error TS2638: Type 'NonNulla
             window.ontouchstart
         } else {
             window.ontouchstart
-                   ~~~~~~~~~~~~

```

Comment:
if you decide to land this PR then I think that it would best to adjust the "title" of this test - the containing function is named `negativeIntersectionTest` and that name was targeting the previous behavior